src/lib/polymer-bootstrap.html

```diff
@@ -37,13 +37,28 @@
       var options = {
         prototype: prototype
       };
-      // NOTE: we're specifically supporting older Chrome versions here 
+      // NOTE: we're specifically supporting older Chrome versions here
       // (specifically Chrome 39) that throw when options.extends is undefined.
       if (prototype.extends) {
         options.extends = prototype.extends;
       }
-      Polymer.telemetry._registrate(prototype);
-      document.registerElement(prototype.is, options);
+
+      try {
```

Comment:
Keep in mind that every try/catch doesn't necessarily go through deopt. This piece of code would also need to be optimized in the first place for it to deopt. Registering elements only happens at boot and very few times. It's like this would never get optimized in the first place.